README.md

```diff
@@ -99,4 +106,23 @@ Landscape provides 5 built-in widgets:
 
 All of them are enabled by default. You can edit them in `widget` setting.
 
-[Fancybox]: https://github.com/fancyapps/fancyBox
\ No newline at end of file
+### Social Links
+
+You can add your social media links like this:
+
+```
+social:
+  github: https://github.com/your_github_account
+  twitter: https://twitter.com/your_twitter_account
+  telegram: https://t.me/youryour_telegram_account
+```
+
+They will appear at the top right conner of the page alongside rss feeds and search icon. Landscape use fontawesome version [*4.7.0*](https://fontawesome.com/v4.7.0/), and the key in each key-value pair is used to find the icon, e.g. github means `<span class="fa fa-github"></span>`, the value is the link. Your can search for avalilable icons and their names [here](https://fontawesome.com/v4.7.0/icons/). Note that the twitter-id you set at top level of _config.yml like:
```

Comment:
```
Specified links will appear at the top right corner of the page,
alongside the rss and search icons.
Each name must correspond to the icon name of [Font Awesome 4](https://fontawesome.com/v4.7.0/icons/).
```

Merge the above into a single line. I don't think twitter id needs to be mentioned since it's clear that the setting is different.